code/game/objects/items/storage/backpack.dm

```diff
@@ -858,11 +858,17 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 	var/camo_alpha = 10
 	var/allow_gun_usage = FALSE
 	var/cloak_cooldown
+	var/allowed_stealth_shooting = FALSE
```

Comment:
This var seems redundant as allow_gun_usage already allows you to shoot stealthed.